src/buildtools/FsLexYacc.proj

```diff
@@ -0,0 +1,23 @@
+<Project>
```

Comment:
I'm a bit concerned that we might be re-ingesting FsLex and FsYacc into this codebase.  It took a huge effort to get these tools out of the F# repository and into a separate package....

It's ok to have a 100% verbatim copy of these as part of buildfrom ource but please make sure we don't change or diverge in any way from http://github.com/fsprojects/FsLexYacc or the nuget package.

If there's some way to ensure this I'd be really grateful.